package.json

```diff
@@ -88,12 +88,12 @@
   },
   "scripts": {
     "webpack:serve": "webpack-dev-server --config webpack.dev.js",
-    "django:serve": "python manage.py runserver",
+    "django:serve": "python manage.py runserver 0.0.0.0:8000",
```

Comment:
I'm guessing this will break local non-Docker dev?

Reply:
Again, a quick change of convenience. We can easily get around these two things breaking things by making distinct Docker yarn scripts.

Reply:
So, `0.0.0.0` will bind to localhost as well as outside interfaces; binding to `0.0.0.0` won't break things. But it will make them brittle, as we're inlining the port there.